src/cursor-doc/paredit.ts

```diff
@@ -671,7 +672,9 @@ export function backwardSlurpSexp(
   cursor.backwardList();
   const tk = cursor.getPrevToken();
   if (tk.type == 'open') {
-    const offset = cursor.clone().previous().offsetStart;
+    const previous = cursor.clone().previous();
+    assertIsDefined(previous, 'Expected a token to be before the cursor!');
+    const offset = previous.offsetStart;
```

Comment:
I'm not sure I am reading this correctly. But generally in Paredit and token cursor code we need to gracefully  handle when the structure does not match some expectation. We most often leave things unchanged in those cases. Does this change mean that we croak? I realize that that would be happening before the change as well, I'm more wondering about what we _should_ be doing here...